src/trip/tripPage.jsx

```diff
@@ -1,9 +1,55 @@
 import React from "react"
+import TripsTable from './components/tripsTable'
+import { Grid } from 'semantic-ui-react'
 
 let TripPage = () => {
+
+    const DATA = [
+        {
+            driver: {"username": "Tanguy"},
+            start: new Date(),
+            adress: "Paris 18",
+            type: "BACK",
+            seats: 3,
+            "passengers": [{"username": "Baptiste"}, {"username": "Romain"}, {"username": "Othmane"}]
+        },
+        {
+            driver: {"username": "Momo"},
+            start: new Date('01 Jan 1970 00:45:00 GMT'),
+            adress: "Paris 18",
+            type: "BACK",
+            seats: 4,
+            "passengers": [{"username": "Julie"}, {"username": "Hugo"}]
+        },
+        {
+            driver: {"username": "Patrick"},
+            start: new Date('28 Jan 1979 15:30:00 GMT'),
+            adress: "Montélimart",
+            type: "FORTH",
+            seats: 1,
+            "passengers": [{"username": "Léa"}]
+        },
+        {
+            driver: {"username": "Patrick"},
+            start: new Date('05 Jan 1970 00:00:00 GMT'),
+            adress: "Châlons-sur-Saone",
+            type: "BACK",
+            seats: 6,
+            passengers: []
+        }
+    ]
+
     return (<>
```

Comment:
What's the interest of wrapping the grip in a fragment ?

Reply:
No interest, i was testing the behaviour of fragments vs templates (in vueJS), forgot to delete it